media/css/firefox/nothing-personal/_header.scss

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+.c-page-header {
+    background-color: $browser-background;
+    border-bottom: $border-black;
+
+    @media (prefers-reduced-motion: no-preference){
+        position: sticky;
+        top: 0;
+        z-index: 15;
+    }
+}
+
+.c-page-header-logo-fx {
+    display: flex;
+    justify-content: center;
```

Comment:
Not sure if EOLed OS are any concern, just reporting how it looks with the unsupported CTA included:

![Screen Shot 2024-07-24 at 14 55 53](https://github.com/user-attachments/assets/51da98bf-cc1f-46b3-991f-c2e6583b2810)

(if, maybe just for this campaign page, you'd want to hide the fx-unsupported-message > p not to get in the way…)

Reply:
thanks for pointing this out! it's been fixed